src/restruct_poc/beams/beams.hpp

```diff
@@ -31,24 +31,24 @@ struct Beams {
     size_t max_elem_nodes;
     size_t max_elem_qps;
 
-    Kokkos::View<ElemIndices*> elem_indices;   // View of element node and qp indices into views
-    Kokkos::View<size_t*> node_state_indices;  // State row index for each node
+    Kokkos::View<ElemIndices*> elem_indices;    // View of element node and qp indices into views
+    Kokkos::View<size_t**> node_state_indices;  // State row index for each node
```

Comment:
Looks like the comments need to be updated throughout? Would be nice to add a little introduction to the `Beams` class as well.

Reply:
Good Idea.  I'll do that